src/anbox/cmds/session_manager.cpp

```diff
@@ -106,6 +106,12 @@ anbox::cmds::SessionManager::SessionManager(const BusFactory &bus_factory)
   flag(cli::make_flag(cli::Name{"window-size"},
                       cli::Description{"Size of the window in single window mode, e.g. --window-size=1024,768"},
                       window_size_));
+  flag(cli::make_flag(cli::Name{"standalone"},
+                      cli::Description{"Prevents starting the Container Manager and default container (Experimental)"},
```

Comment:
Actually its more "Prevents starting the Android container via the container manager" as its not really starting the container-manager, it's already started by systemd.

Experimental -> experimental